### YamlMime:ManagedReference
items:
- uid: System.Web.UI.WebControls.EntityDataSourceValidationException
  id: EntityDataSourceValidationException
  children:
  - System.Web.UI.WebControls.EntityDataSourceValidationException.#ctor
  - System.Web.UI.WebControls.EntityDataSourceValidationException.#ctor(System.String)
  - System.Web.UI.WebControls.EntityDataSourceValidationException.#ctor(System.String,System.Exception)
  - System.Web.UI.WebControls.EntityDataSourceValidationException.System#Web#DynamicData#IDynamicValidatorException#InnerExceptions
  langs:
  - csharp
  name: EntityDataSourceValidationException
  nameWithType: EntityDataSourceValidationException
  fullName: System.Web.UI.WebControls.EntityDataSourceValidationException
  type: Class
  summary: "동적 데이터 원본의 속성 유효성을 검사할 때 발생 하는 오류를 나타냅니다."
  syntax:
    content: 'public sealed class EntityDataSourceValidationException : Exception, System.Web.DynamicData.IDynamicValidatorException'
  inheritance:
  - System.Exception
  implements:
  - System.Web.DynamicData.IDynamicValidatorException
  inheritedMembers: []
  platform:
  - net462
- uid: System.Web.UI.WebControls.EntityDataSourceValidationException.#ctor
  id: '#ctor'
  parent: System.Web.UI.WebControls.EntityDataSourceValidationException
  langs:
  - csharp
  name: EntityDataSourceValidationException()
  nameWithType: EntityDataSourceValidationException.EntityDataSourceValidationException()
  fullName: System.Web.UI.WebControls.EntityDataSourceValidationException.EntityDataSourceValidationException()
  type: Constructor
  assemblies:
  - System.Web.Entity
  namespace: System.Web.UI.WebControls
  summary: "새 인스턴스를 만듭니다는 <xref href=&quot;System.Web.UI.WebControls.EntityDataSourceValidationException&quot;> </xref> 오류를 설명 하는 시스템 제공 메시지로 클래스입니다."
  remarks: "이 생성자는 <xref:System.Exception.Message%2A>&quot;defaultmessagedisplayedbyparameterlessctorwritermustsupply&quot;와 같은 오류를 설명 하는 시스템 제공 메시지로 새 인스턴스의 속성.</xref:System.Exception.Message%2A> 이 메시지는 현재 시스템 문화권을 고려 합니다.       다음 표에서 <xref:System.InsufficientMemoryException>안내 하십시오.</xref:System.InsufficientMemoryException> 인스턴스에 대 한 초기 속성 값을 보여 줍니다.      | 속성 | 값 |   |--------------|-----------|   | <xref:System.Exception.InnerException%2A>| `null`.|   | <xref:System.Exception.Message%2A>| 지역화 된 오류 메시지 문자열입니다. |</xref:System.Exception.Message%2A> </xref:System.Exception.InnerException%2A>"
  syntax:
    content: public EntityDataSourceValidationException ();
    parameters: []
  overload: System.Web.UI.WebControls.EntityDataSourceValidationException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Web.UI.WebControls.EntityDataSourceValidationException.#ctor(System.String)
  id: '#ctor(System.String)'
  parent: System.Web.UI.WebControls.EntityDataSourceValidationException
  langs:
  - csharp
  name: EntityDataSourceValidationException(String)
  nameWithType: EntityDataSourceValidationException.EntityDataSourceValidationException(String)
  fullName: System.Web.UI.WebControls.EntityDataSourceValidationException.EntityDataSourceValidationException(String)
  type: Constructor
  assemblies:
  - System.Web.Entity
  namespace: System.Web.UI.WebControls
  summary: "새 인스턴스를 만듭니다는 <xref href=&quot;System.Web.UI.WebControls.EntityDataSourceValidationException&quot;> </xref> 지정 된 메시지 오류를 설명 하는 클래스입니다."
  remarks: "다음 표에서 <xref:System.Web.UI.WebControls.EntityDataSourceValidationException>안내 하십시오.</xref:System.Web.UI.WebControls.EntityDataSourceValidationException> 인스턴스에 대 한 초기 속성 값을 보여 줍니다.      | 속성 | 값 |   |--------------|-----------|   | <xref:System.Exception.InnerException%2A>| `null`.|   | <xref:System.Exception.Message%2A>| 에 지정 된 오류 메시지 문자열 `message`. |</xref:System.Exception.Message%2A> </xref:System.Exception.InnerException%2A>"
  syntax:
    content: public EntityDataSourceValidationException (string message);
    parameters:
    - id: message
      type: System.String
      description: "예외를 설명 하는 메시지입니다. 이 생성자의 호출자는이 문자열이 현재 시스템 문화권에 맞게 지역화 되었는지 확인 해야 합니다."
  overload: System.Web.UI.WebControls.EntityDataSourceValidationException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Web.UI.WebControls.EntityDataSourceValidationException.#ctor(System.String,System.Exception)
  id: '#ctor(System.String,System.Exception)'
  parent: System.Web.UI.WebControls.EntityDataSourceValidationException
  langs:
  - csharp
  name: EntityDataSourceValidationException(String,Exception)
  nameWithType: EntityDataSourceValidationException.EntityDataSourceValidationException(String,Exception)
  fullName: System.Web.UI.WebControls.EntityDataSourceValidationException.EntityDataSourceValidationException(String,Exception)
  type: Constructor
  assemblies:
  - System.Web.Entity
  namespace: System.Web.UI.WebControls
  summary: "새 인스턴스를 만듭니다는 <xref href=&quot;System.Web.UI.WebControls.EntityDataSourceValidationException&quot;> </xref> 지정된 된 오류 메시지와 해당 예외의 원인인 내부 예외에 대 한 참조를 사용 하 여 클래스입니다."
  remarks: "이전 예외의 직접적인 결과에 이전 예외에 대 한 참조가 들어 있어야 throw 되는 예외는 <xref:System.Exception.InnerException%2A>속성.</xref:System.Exception.InnerException%2A> <xref:System.Exception.InnerException%2A>속성은 생성자에 전달 되는 동일한 값을 반환 하거나 `null` 경우는 <xref:System.Exception.InnerException%2A>생성자에 내부 예외 값을 제공 하지 않는.</xref:System.Exception.InnerException%2A> </xref:System.Exception.InnerException%2A>       다음 표에서 <xref:System.Web.UI.WebControls.EntityDataSourceValidationException>안내 하십시오.</xref:System.Web.UI.WebControls.EntityDataSourceValidationException> 인스턴스에 대 한 초기 속성 값을 보여 줍니다.      | 속성 | 값 |   |--------------|-----------|   | <xref:System.Exception.InnerException%2A>| `null`.|   | <xref:System.Exception.Message%2A>| 에 지정 된 오류 메시지 문자열 `message`. |</xref:System.Exception.Message%2A> </xref:System.Exception.InnerException%2A>"
  syntax:
    content: public EntityDataSourceValidationException (string message, Exception innerException);
    parameters:
    - id: message
      type: System.String
      description: "예외를 설명 하는 메시지입니다. 이 생성자의 호출자는이 문자열이 현재 시스템 문화권에 맞게 지역화 되었는지 확인 해야 합니다."
    - id: innerException
      type: System.Exception
      description: "현재 예외의 원인인 예외입니다. 경우는 `innerException` 매개 변수가 않습니다 <xref uid=&quot;langword_csharp_null&quot; name=&quot;null&quot; href=&quot;&quot;> </xref>에서 현재 예외가 발생 한 <xref uid=&quot;langword_csharp_catch&quot; name=&quot;catch&quot; href=&quot;&quot;> </xref> 내부 예외를 처리 하는 블록입니다."
  overload: System.Web.UI.WebControls.EntityDataSourceValidationException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Web.UI.WebControls.EntityDataSourceValidationException.System#Web#DynamicData#IDynamicValidatorException#InnerExceptions
  id: System#Web#DynamicData#IDynamicValidatorException#InnerExceptions
  isEii: true
  parent: System.Web.UI.WebControls.EntityDataSourceValidationException
  langs:
  - csharp
  name: System.Web.DynamicData.IDynamicValidatorException.InnerExceptions
  nameWithType: EntityDataSourceValidationException.System.Web.DynamicData.IDynamicValidatorException.InnerExceptions
  fullName: System.Web.UI.WebControls.EntityDataSourceValidationException.System.Web.DynamicData.IDynamicValidatorException.InnerExceptions
  type: Property
  assemblies:
  - System.Web.Entity
  namespace: System.Web.UI.WebControls
  summary: "새로 만들거나 편집한 데이터 필드의 유효성을 검사할 때 발생 하는 예외를 가져옵니다."
  remarks: "이 멤버는 명시적 인터페이스 멤버 구현 이며 사용할 수 있습니다 경우에만 <xref:System.Web.UI.WebControls.EntityDataSourceValidationException>인스턴스로 캐스팅 되는 <xref:System.Web.DynamicData.IDynamicValidatorException>인터페이스.</xref:System.Web.DynamicData.IDynamicValidatorException> </xref:System.Web.UI.WebControls.EntityDataSourceValidationException>"
  syntax:
    content: System.Collections.Generic.IDictionary<string,Exception> System.Web.DynamicData.IDynamicValidatorException.InnerExceptions { get; }
    return:
      type: System.Collections.Generic.IDictionary{System.String,System.Exception}
      description: "발생 한 모든 예외를 포함 하는 컬렉션입니다."
  overload: System.Web.UI.WebControls.EntityDataSourceValidationException.System#Web#DynamicData#IDynamicValidatorException#InnerExceptions*
  exceptions: []
  platform:
  - net462
references:
- uid: System.Exception
  isExternal: false
  name: System.Exception
- uid: System.Web.UI.WebControls.EntityDataSourceValidationException.#ctor
  parent: System.Web.UI.WebControls.EntityDataSourceValidationException
  isExternal: false
  name: EntityDataSourceValidationException()
  nameWithType: EntityDataSourceValidationException.EntityDataSourceValidationException()
  fullName: System.Web.UI.WebControls.EntityDataSourceValidationException.EntityDataSourceValidationException()
- uid: System.Web.UI.WebControls.EntityDataSourceValidationException.#ctor(System.String)
  parent: System.Web.UI.WebControls.EntityDataSourceValidationException
  isExternal: false
  name: EntityDataSourceValidationException(String)
  nameWithType: EntityDataSourceValidationException.EntityDataSourceValidationException(String)
  fullName: System.Web.UI.WebControls.EntityDataSourceValidationException.EntityDataSourceValidationException(String)
- uid: System.String
  parent: System
  isExternal: true
  name: String
  nameWithType: String
  fullName: System.String
- uid: System.Web.UI.WebControls.EntityDataSourceValidationException.#ctor(System.String,System.Exception)
  parent: System.Web.UI.WebControls.EntityDataSourceValidationException
  isExternal: false
  name: EntityDataSourceValidationException(String,Exception)
  nameWithType: EntityDataSourceValidationException.EntityDataSourceValidationException(String,Exception)
  fullName: System.Web.UI.WebControls.EntityDataSourceValidationException.EntityDataSourceValidationException(String,Exception)
- uid: System.Web.UI.WebControls.EntityDataSourceValidationException.System#Web#DynamicData#IDynamicValidatorException#InnerExceptions
  parent: System.Web.UI.WebControls.EntityDataSourceValidationException
  isExternal: false
  name: System.Web.DynamicData.IDynamicValidatorException.InnerExceptions
  nameWithType: EntityDataSourceValidationException.System.Web.DynamicData.IDynamicValidatorException.InnerExceptions
  fullName: System.Web.UI.WebControls.EntityDataSourceValidationException.System.Web.DynamicData.IDynamicValidatorException.InnerExceptions
- uid: System.Collections.Generic.IDictionary{System.String,System.Exception}
  parent: System.Collections.Generic
  isExternal: true
  name: IDictionary<String,Exception>
  nameWithType: IDictionary<String,Exception>
  fullName: System.Collections.Generic.IDictionary<System.String,System.Exception>
  spec.csharp:
  - uid: System.Collections.Generic.IDictionary`2
    name: IDictionary
    nameWithType: IDictionary
    fullName: IDictionary<System.String,System.Exception>
  - name: <
    nameWithType: <
    fullName: <
  - uid: System.String
    name: String
    nameWithType: String
    fullName: String
  - name: ','
    nameWithType: ','
    fullName: ','
  - uid: System.Exception
    name: Exception
    nameWithType: Exception
    fullName: Exception
  - name: '>'
    nameWithType: '>'
    fullName: '>'
- uid: System.Web.UI.WebControls.EntityDataSourceValidationException.#ctor*
  parent: System.Web.UI.WebControls.EntityDataSourceValidationException
  isExternal: false
  name: EntityDataSourceValidationException
  nameWithType: EntityDataSourceValidationException.EntityDataSourceValidationException
- uid: System.Web.UI.WebControls.EntityDataSourceValidationException.System#Web#DynamicData#IDynamicValidatorException#InnerExceptions*
  parent: System.Web.UI.WebControls.EntityDataSourceValidationException
  isExternal: false
  name: System.Web.DynamicData.IDynamicValidatorException.InnerExceptions
  nameWithType: EntityDataSourceValidationException.System.Web.DynamicData.IDynamicValidatorException.InnerExceptions
